experimenter/tests/integration/nimbus/ios/xcodebuild.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import subprocess
+from pathlib import Path
+
+from .xcrun import XCRun
+
+here = Path(__file__).resolve()
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class XCodeBuild:
+    device = os.getenv("IOS_DEVICE", "iPhone 16")
+    ios_version = os.getenv("IOS_VERSION", "18.1")
+    binary = "xcodebuild"
+    destination = f"platform=iOS Simulator,name={device},OS={ios_version}"
+    logger = logging.getLogger()
+    scheme = "Fennec"
+    testPlan = "SyncIntegrationTestPlan"
+    xcrun = XCRun()
+
+    def __init__(self, log, **kwargs):
+        self.scheme = kwargs.get("scheme", self.scheme)
+        self.testPlan = kwargs.get("test_plan", self.testPlan)
+        self.log = log
+        self.firefox_app_path = next(
+            Path("/Users").glob(
+                "**/Library/Developer/Xcode/DerivedData/Client-*/Build/Products/Fennec_Testing-*/Client.app"
+            )
+        )
+
+    def install(self, boot=True):
+        if boot:
+            self.xcrun.boot()
+        try:
+            out = subprocess.check_output(
+                f"xcrun simctl install booted {self.firefox_app_path}",
+                cwd=f"{here.parent}",
+                stderr=subprocess.STDOUT,
+                universal_newlines=True,
+                shell=True,
+            )
```

Comment:
if you use shell=True you have to worry about shell escaping/sanitization of firefox_app_path

```suggestion
            out = subprocess.check_output(
                ["xcrun", "simctl", "install", "booted", self.firefox_app_path],
                cwd=f"{here.parent}",
                stderr=subprocess.STDOUT,
                universal_newlines=True,
            )
```